docs/source/algorithms/MaxEnt-v1.rst

```diff
@@ -526,6 +527,183 @@ Output:
+       for i in range(0,N):
+           X.append(X[i])
+           Ylin.append(s*Magnitude*X[i])
+           Zeroes.append(0)
+        
+   CreateWorkspace(OutputWorkspace='linadj', DataX=X, DataY=Ylin, DataE=Zeroes, NSpec=6)
+
+   # Construct linear adjustment workspace (real = 0, imaginary linear)
+   # no adjustment on first spectrum, double adjustment on third spectrum.
+   Yconst = []
+   Magnitude = 0.2
+   # Real values
+   for s in range(0,3):
+       for i in range(0,N):
+           Yconst.append(0)
+        
+   # Imaginary values
+   for s in range(0,3):
+       for i in range(0,N):
+           Yconst.append(s*Magnitude*X[i])
+        
+   CreateWorkspace(OutputWorkspace='constadj',DataX=X, DataY=Yconst, DataE=Zeroes, NSpec=6)
+
+   evolChi, evolAngle, image, data = MaxEnt(InputWorkspace='inputws', DataLinearAdj='linadj', DataConstAdj='constadj',A=0.001)
+    
+   print("Reconstruction at 05 of first spectrum: {:.3f}".format(data.readY(0)[5]))
+   print("Reconstruction at 10 of first spectrum: {:.3f}".format(data.readY(0)[10]))
+   print("Reconstruction at 15 of first spectrum: {:.3f}".format(data.readY(0)[15]))
+   print("Reconstruction at 05 of third spectrum: {:.3f}".format(data.readY(2)[5]))
+   print("Reconstruction at 10 of third spectrum: {:.3f}".format(data.readY(2)[10]))
+   print("Reconstruction at 15 of third spectrum: {:.3f}".format(data.readY(2)[15]))
+   print("Number of iterations of first spectrum: "+str( len(evolAngle.readX(0))))
+   print("Number of iterations of second spectrum: "+str( len(evolAngle.readX(1))))
+   print("Number of iterations of third spectrum: "+str( len(evolAngle.readX(2))))
+
+Output:
+
+.. testoutput:: ExAdjustment
+   
+   Reconstruction at 05 of first spectrum: 0.657
+   Reconstruction at 10 of first spectrum: 0.433
+   Reconstruction at 15 of first spectrum: 0.115
+   Reconstruction at 05 of third spectrum: 0.522
+   Reconstruction at 10 of third spectrum: 0.485
+   Reconstruction at 15 of third spectrum: -0.019
+   Number of iterations of first spectrum: 5
+   Number of iterations of second spectrum: 37
+   Number of iterations of third spectrum: 70
+   
+.. figure:: ../images/MaxEntAdjust.png
+   :align: center
+   
+   The first plot shows the reconstructed data and the second plot the corresponding image.
+   The first three spectra plotted are the real parts of the 3 spectra
+   and the other three spectra are the imaginary parts.
+   
+This can also be done with the spectra in the reconstructed data and image summed together. 
+This is done by setting the *PerSpectrumReconstruction* property to *false*.
+
+.. testcode:: ExAdjustmentSummed
+
+   from math import pi, sin, cos
+   from random import random, seed
+
+   # Construct workspace (cosine + noise) over 3 spectra
+   X = []
+   Y = []
+   E = []
+   N = 200
+   w = 3
+   for s in range(0,3):
+      seed(0)
+      for i in range(0,N):
+        x = 2*pi*i/N
+        X.append(x)
+        Y.append(cos(w*x)+(random()-0.5)*0.3)
```

Comment:
to demonstrate the linear adjustment would it not be better to create a workspace that is offset by a vector of constants (e.g. `cos(w*x)+x` ) then have the `Xlin` as having `Ylin.append(X[i])`? Similarly for multiply

Reply:
for ``Xlin``?

Reply:
line 659

Reply:
I would like to try this, but do not understand what you are suggesting.

Reply:
you would have ```offset = [1,2,3,4]``` then your y data would be ```y[i] = cos(w*x[i])+offset``` Then the ```DataLinearAdj``` would just be a workspace with the y values from offset